validated/tables.py

```diff
@@ -0,0 +1,287 @@
+from dash_ag_grid import AgGrid
+
+
+def create_daily_table(data: dict) -> AgGrid:
+    """Creates Daily Report table
+
+    Args:
+        data (dict): Daily report data (from functions.daily_validation)
+
+    Returns:
+        AgGrid: Daily report table
+    """
+    table = AgGrid(
+        id="table_daily",
+        rowData=data["data"],
+        columnDefs=create_columns_daily(value_columns=data["value_columns"]),
+        columnSize="sizeToFit",
+        defaultColDef={
+            "resizable": True,
+            "sortable": True,
+            "checkboxSelection": {
+                "function": "params.column == params.columnApi.getAllDisplayedColumns()[0]"
```

Comment:
Where is this `params` object coming from? Or, in other words, how this `checkboxSelection` business work?